Resources/config/form.config.xml

```diff
@@ -10,21 +10,21 @@
             <argument type="service" id="integrated_channel.form.config.type" />
             <argument type="string">channel_config_new</argument>
 
-            <tag name="form.type" alias="channel_config_new" />
+            <tag name="form.type" />
```

Comment:
@jansanne  this XML is  removed, but I have also removed the parameters from the form.xml.